src/main/java/appeng/helpers/DualityInterface.java

```diff
@@ -880,14 +891,12 @@ public IStorageMonitorable getMonitorable( final IActionSource src, final IStora
 		{
 
 			@Override
-			public IMEMonitor<IAEItemStack> getItemInventory()
-			{
-				return new InterfaceInventory( di );
-			}
-
-			@Override
-			public IMEMonitor<IAEFluidStack> getFluidInventory()
+			public <T extends IAEStack<T>, C extends IStorageChannel<T>> IMEMonitor<T> getInventory( IStorageChannel<T> channel )
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') C is not used in the method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS2326)